app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -17,8 +17,6 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     end
 
     api :GET, "/organizations/:organization_id/subscriptions", N_("List organization subscriptions")
-    api :GET, "/systems/:system_id/subscriptions", N_("List a content host's subscriptions"), :deprecated => true
-    api :GET, "/activation_keys/:activation_key_id/subscriptions", N_("List an activation key's subscriptions")
```

Comment:
If you mean how data is collected from candlepin, the changes with systems scoped search shouldn't affect that.